modules/ROOT/pages/features/MediaRecordingOnDemand.adoc

```diff
@@ -24,12 +24,144 @@ recording:
   filters:
     - filter: "ip.src_addr=26.03.19.93 sip.user=desi"                         // <1>
       mode : 0                                                                // <2>
-    - filter: "ip.host=RTP_ENGINE_1 media.duration>10000 media.r_factor<75"   // <3>
+    - filter: "ip.host=RTP_ENGINE_1 media.r_factor<75 media.duration>10000"   // <3>
       mode : 1                                                                // <4>
 
 ----
 
-<1> - SIP signaling filter. It may contain next list of attributes: `ip.src_addr`, `ip_dst_addr`, `ip.addr`, `sip.caller`, `sip.calle`, `sip.user`.
-<2> - `FULL` mode. In this mode SIP3 will record full RTP packets including voice payload.
-<3> - Media quality filter. It may contain next list of attributes: `ip.src_addr`, `ip_dst_addr`, `ip.addr`, `media.duration`, `media.mos`, `media.r_factor`.
-<4> - `GDRP` mode. In this mode SIP3 will record only headers of RTP packets without voice payload.
+<1> SIP signaling filter that enables recording for session from `26.03.19.93` to or from user `desi`.
+<2> `FULL` mode. In this mode SIP3 will record full RTP packets including voice payload.
+<3> Media quality filter that enables recording for Media sessions with host `RTP_ENGINE_1`, R-Factor less that 75 and duration of media more than 10000ms (10 seconds)
+<4> `GDPR` mode. In this mode SIP3 will record only headers of RTP packets without voice payload.
+
+Now more about filters.
+
+== Filters
+
+There are two types of filters in `SIP3` for `Media Recording On Demand`:
+
+1. `SIP signaling filter` - applying at Codec negotiation.
+2. `Media quality filter` - applying on each periodic RTP Report handled by `Salto`.
+
+Configuration of `Media Recording On Demand` may contain multiple filters both of types. All filters will be loaded to `SIP3` and checking in order from configuration. The first matching filter's recording mode will be applied to matched Call/Media session.
+
+=== Modes
+
+Each filter controls Media Recording Mode. SIP3 supports `FULL` and `GDPR` modes:
+
+[%header,cols=3*]
+|===
+|Mode |Value |Description
+
+|`FULL`
+|`0`
+|Full RTP/RTCP and related ICMP packets including voice payload
+
+|`GDPR`
+|`1`
+|Only headers of RTP/RTCP and related ICMP packets without voice payload.
+|===
+
+In addition to GDPR compliance of `GDPR` mode requires much less storage space (about 3-4 times less).
```

Comment:
In addition to GDPR compliance, `GDPR` mode requires 3-4 times less storage space.